src/woo_publications/metadata/models.py

```diff
@@ -20,7 +20,7 @@ class InformationCategory(OrderedModel):
     identifier = models.URLField(
         _("identifier"),
         help_text=_(
-            "The unique IRI that identifies this category in the overheid.nl value list. "
+            "The unique URI that identifies this category in the overheid.nl value list. "
```

Comment:
Please revert this. An IRI is more specific than URI - it allows unicode charachters where a URI is limited to ASCII